index.js

```diff
@@ -12,31 +15,34 @@ module.exports = {
    * http://clang.llvm.org/docs/ClangFormatStyleOptions.html#configurable-format-style-options
    */
   checkFormat: function(options) {
-    options = (typeof options === "undefined") ?
-                  {
-                    BasedOnStyle: "Google"
-                  } :
-                  options;
+    options =
+        (typeof options === "undefined") ? {BasedOnStyle: "Google"} : options;
     var optsStr =
         (typeof options === "string") ? options : JSON.stringify(options);
```

Comment:
Why can options be a string or a JSON object or undefined? That's a bit confusing. If that really must be, could you move that to if/else statements to make the different possible combinations clearer?